cmd/osquery-perf/agent.go

```diff
@@ -723,9 +723,10 @@ func loadSoftware(platform string, ver string) []map[string]string {
 	var r []map[string]string
 	for _, fi := range software {
 		r = append(r, map[string]string{
-			"name":    fi.Name,
-			"version": fi.Version,
-			"source":  "osquery-perf",
+			"name":           fi.Name,
+			"version":        fi.Version,
+			"source":         "osquery-perf",
+			"installed_path": fmt.Sprintf("/some/path/%s", fi.Name),
```

Comment:
Done in https://github.com/fleetdm/fleet/pull/11214/commits/a1e6a7fec5016ccc6450e831963acf9892bc69df

Reply:
OK, looks good, double checking that this will be setting "" for the same software every time (to be more realistic to the real world). It does seem like that because it loads the software from the same JSON array every time.

Reply:
Yeah it should - like you mentioned the JSON array is processed sequentially